packages/auto-approve/src/process-checks/update-discovery-artifacts.ts

```diff
@@ -63,7 +61,6 @@ export class UpdateDiscoveryArtifacts extends Process implements LanguageRule {
       (this.classRule = {
         author: 'yoshi-code-bot',
         titleRegex: /^chore: Update discovery artifacts/,
-        maxFiles: 2,
```

Comment:
@parthea, saw a bug with the update-discovery-artifacts process, currently it caps the amount of files changed to 2, but your PRs are usually much bigger. I removed it in this PR.